cli/args/config_file.rs

```diff
@@ -612,6 +651,14 @@ pub struct BenchConfig {
   pub files: FilesConfig,
 }
 
+impl BenchConfig {
+  pub fn extend_files(&self, files: &FilesConfig) -> Self {
+    BenchConfig {
+      files: self.files.clone() + files.clone(),
```

Comment:
Why would someone be adding the same file to both bench and test? I'm not sure it makes sense to have "include"

Reply:
ok, i'll remove the `include` field.